corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -73,6 +73,10 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
             'session_id',
         ],
 
+        queryProperties: [
```

Comment:
this would not exist in the check on line 158 in `controller.js` because that would be checking for an entities  response-- so this would need to be available for query and entities responses

Reply:
I made some changes so the lines are shifted but I think you mean these line:
```
if (Object.keys(queryResponse.groupHeaders).length > 0) {
    queryCollection = new Collection(menusUtils.groupDisplays(queryResponse.displays, queryResponse.groupHeaders));
}
```
`queryResponse` is part of the entities response and does get copied. So this works in both cases.